armory/scenarios/carla_object_detection.py

```diff
@@ -30,11 +30,11 @@ def load_dataset(self):
 
     def next(self):
         super().next()
-        self.y_object = [self.y[0]]
         self.y_patch_metadata = [self.y[1]]
+        self.y = [self.y[0]]
```

Comment:
Is y a 2-tuple? If so, it might be good to show that we are not throwing anything away.
I.e., 
```
self.y, self.y_patch_metadata = [[y_i] for y_i in self.y]
```